firebase/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+''' Builds and exports Firebase dependencies.
```

Comment:
Suggest: Package for all Firebase dependencies.

Reply:
Done!